k9mail/src/main/java/com/fsck/k9/notification/NotificationHolder.java

```diff
@@ -0,0 +1,13 @@
+package com.fsck.k9.notification;
+
+
+class NotificationHolder {
```

Comment:
I went with `NotificationData` mostly because that's what the class that contained most of the functionality of the current class was called before.